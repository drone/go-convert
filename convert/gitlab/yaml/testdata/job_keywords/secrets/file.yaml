# https://docs.gitlab.com/ee/ci/yaml/#secretsfile

job:
  secrets:
    DATABASE_PASSWORD:
      vault: production/db/password@ops
      file: false