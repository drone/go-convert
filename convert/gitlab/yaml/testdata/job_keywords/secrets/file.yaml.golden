job:
    secrets:
        DATABASE_PASSWORD:
            vault:
                engine:
                    name: kv-v2
                    path: ops
                path: production/db
                field: password
            file: false